src/test/scala/org/broadinstitute/dsde/agora/server/webservice/AgoraMethodsSpec.scala

```diff
@@ -107,11 +105,16 @@ class AgoraMethodsSpec extends ApiServiceSpec {
   }
 
   "Agora" should "store 10kb of github markdown as method documentation and return it without alteration" in {
-    Post(ApiUtil.Methods.withLeadingSlash, testAgoraEntityBigDoc) ~>
+    Post(ApiUtil.Methods.withLeadingSlash + "?includedField=documentation", testAgoraEntityBigDoc) ~>
       methodsService.postRoute ~> check {
       handleError(
         entity.as[AgoraEntity],
-        (entity: AgoraEntity) => assert(entity.documentation.get === bigDocumentation.get)
+        (entity: AgoraEntity) => {
+          val searchEntity = new AgoraEntity(name = entity.name, namespace = entity.namespace, snapshotId = entity.snapshotId)
```

Comment:
Ah good points.
The issue is that findSingle is used both on methods?namespace?name?id and on the post so will get one behavior or the other for both methods since the converge here.

I'm going to revert changes to findSingle since I think that use case of getting the entire entity is more important.